In [2]:
import pandas as pd
import requests
from bs4 import BeautifulSoup
import numpy as np

In [8]:
requests.get('https://www.ambitionbox.com/list-of-companies?page=1')

<Response [403]>

#### if response code is 403
- headers={'User-Agent':'Mozilla/5.0 (Windows NT 6.3; Win 64 ; x64) Apple WeKit /537.36(KHTML , like Gecko) Chrome/80.0.3987.162 Safari/537.36'} 
- requests.get('url',headers=headers).text

In [11]:
requests.get('https://www.ambitionbox.com/list-of-companies?page=1').text

'<HTML><HEAD>\n<TITLE>Access Denied</TITLE>\n</HEAD><BODY>\n<H1>Access Denied</H1>\n \nYou don\'t have permission to access "http&#58;&#47;&#47;www&#46;ambitionbox&#46;com&#47;list&#45;of&#45;companies&#63;" on this server.<P>\nReference&#32;&#35;18&#46;ad0ec417&#46;1734948670&#46;ff59b2a\n<P>https&#58;&#47;&#47;errors&#46;edgesuite&#46;net&#47;18&#46;ad0ec417&#46;1734948670&#46;ff59b2a</P>\n</BODY>\n</HTML>\n'

In [19]:
url = 'https://www.ambitionbox.com/list-of-companies?page=1'

In [21]:
headers={'User-Agent':'Mozilla/5.0 (Windows NT 6.3; Win 64 ; x64) Apple WeKit /537.36(KHTML , like Gecko) Chrome/80.0.3987.162 Safari/537.36'}

In [25]:
requests.get(url,headers=headers).text

ConnectionError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))